docs/security.rst

```diff
@@ -60,3 +60,70 @@ Girder returns an error when a Cross-Origin request is made (one with the
 ``Origin`` header) that does not match the system configuration settings.
 Although most modern web browsers also enforce this, some additional security
 is added by enforcing it at the request level.
+
+
+Database Injection Attacks
+--------------------------
+
+Girder defends against database injection attacks by using PyMongo as the only
```

Comment:
I don't think this paragraph is accurate. It is indeed possible to write injection-vulnerable code with PyMongo. See something like:
def exploitMe(operation, database, collection, data):
    """ This is a super-helpful general-purpose database interaction function. What could possibly go wrong?"""
    getattr(pymongo.MongoClient()[database][collection], operation)(data)

This should read something like (feel free to rephrase):

Girder is defends against database injection attacks by using MongoDB, which is immune to traditional SQL injection attacks. See the <link to Mongo FAQ> for more information on this. Additionally, Girder uses a database model layer to mediate and validate all interaction with the database. This ensures that for all database operations, structural attributes (collection name, operation type) are hardcoded and not modifiable by the client, while data attributes (stored content) are validated for proper form before being accepted from a client.